data/core/tips.json

```diff
@@ -52,7 +52,7 @@
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Afraid of wildlife?  Yell until it goes away.  But beware of what comes instead." ]
+    "text": [ "Noise and fire will scare off a lot of dangerous animals." ]
```

Comment:
Why change this? It is completely valid as originally written, and was less dry.

Reply:
As originally written it was unclear and misleading. No mention of fire and the noise levels that scare animals are low enough to not draw hordes.

Reply:
I'd suggest still mentioning yelling and hordes. Open with `Noise - like shouting -` and close with `Careful, though. Noise can attract the interest of things worse than wildlife.`